pkg/machines/components/vmOverviewTabLibvirt.jsx

```diff
@@ -44,6 +50,17 @@ class VmOverviewTabLibvirt extends React.Component {
         this.state = { showModal: false };
         this.open = this.open.bind(this);
         this.close = this.close.bind(this);
+        this.onAutostartChanged = this.onAutostartChanged.bind(this);
+    }
+
+    onAutostartChanged() {
+        const { dispatch, vm } = this.props;
+        const autostart = !vm.autostart;
+
+        dispatch(changeVmAutostart({ vm, autostart }))
```

Comment:
dispatch is used for dispatching actions which have reducers or use functions that have reducers. This one doesn't have, which honestly is a bad practice that I am following long time now :( (functions that don't update the store with some reducer should not have action creators and reducers) 
All in all I think dispatch is not needed here.